egs/wsj/s5/steps/nnet3/train_dnn.py

```diff
@@ -1,680 +1,417 @@
 #!/usr/bin/env python
 
-
-# Copyright 2016 Vijayaditya Peddinti.
+# Copyright 2016    Vijayaditya Peddinti.
+#           2016    Vimal Manohar
 # Apache 2.0.
 
+""" This script is based on steps/nnet3/tdnn/train.sh
+"""
 
-# this script is based on steps/nnet3/lstm/train.sh
-
-
-import subprocess
 import argparse
-import sys
-import pprint
 import logging
-import imp
+import os
+import pprint
+import shutil
+import sys
 import traceback
-from nnet3_train_lib import *
 
-nnet3_log_parse = imp.load_source('', 'steps/nnet3/report/nnet3_log_parse_lib.py')
+import libs.nnet3.train.common as common_train_lib
+import libs.common as common_lib
+import libs.nnet3.train.frame_level_objf as train_lib
+import libs.nnet3.report.log_parse as nnet3_log_parse
+
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('libs')
```

Comment:
why does just this call to getLogger use the name 'libs'?
BTW, please fix the issues that Gaofeng found (see email).